app/frontend/actions/wallet.ts

```diff
@@ -60,29 +62,48 @@ export default (store: Store) => {
     {
       cryptoProviderType,
       walletSecretDef,
-      forceWebUsb,
+      selectedLedgerTransportType,
```

Comment:
the "selected" prefix seems redundant to me and reflects on a concern irrelevant to the crypto provider (that the transport has been "selected", but that doesn't really matter)

Reply:
It's not passed to crypto provider as `selectedLedgerTransportType`. Variable is named `selectedLedgerTransportType` in the UI, where it reflect's what user selected, after being passed to wallet logic, it's "transformed" to `ledgerTransportType` where the `Default` is changed to specific transport type.